jest.config.js

```diff
@@ -1,29 +1,74 @@
 const config = {
+  cacheDirectory: '<rootDir>/.cache/jest/'
+}
+
+module.exports = {
   projects: [
     {
+      ...config,
+      displayName: 'Gulp tasks',
+      testMatch: [
+        '**/gulp/**/*.test.{js,mjs}'
+      ]
+    },
+    {
+      ...config,
       displayName: 'Snapshot tests',
```

Comment:
Appreciate this name is not new, but it's not what I'd naturally call these. Especially as Jest has its own meaning for 'snapshots'.

Maybe 'Nunjucks macro tests'?

Reply:
```suggestion
      displayName: 'Nunjucks macro tests',
```

Reply:
Updated this one too 👍